Back-end/src/com/smartplanner/GoWorkBruteforcer.java

```diff
@@ -0,0 +1,54 @@
+package com.smartplanner;
+
+import java.util.ArrayList;
+import java.util.concurrent.atomic.AtomicInteger;
+
+public class GoWorkBruteforcer {
```

Comment:
For me, this name isn't descriptive enough.

Reply:
what about GoToWorkDecisionPointBruteforcer?